crates/ruff_cli/src/lib.rs

```diff
@@ -69,6 +69,10 @@ quoting the executed command, along with the relevant file contents and `pyproje
         }));
     }
 
+    // Enabled ANSI colors on Windows 10.
+    #[cfg(windows)]
```

Comment:
This is what I see everyone doing on GitHub Code Search, not sure if we can scope this any more narrowly.